care/facility/tests/test_asset_api.py

```diff
@@ -38,6 +44,16 @@ def test_create_asset(self):
         response = self.client.post("/api/v1/asset/", sample_data)
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
 
+    def test_create_asset_read_only(self):
```

Comment:
@gigincg this would require creating user objects for each of the user type in the test's setup. 

Checking with one of the read-only user types with the highest access level seems to be fine right?